pkg/artifactory/resource/repository/local/local.go

```diff
@@ -142,14 +145,23 @@ var BaseLocalRepoSchema = map[string]*schema.Schema{
 	},
 }
 
+// `packageType` in the API call payload for Terraform repositories must be "terraform", but we use
+// `terraform_module` and `terraform_provider` as a package types in the Provider. getRepoType function corrects this discrepancy.
+var GetPackageType = func(repoType string) string {
```

Comment:
this is top level and it's a var which means it can be reassigned, which you would never want. Just make it a func

Reply:
ok, replaced with a function